backend/src/cmd/import_realm_tree.rs

```diff
@@ -62,31 +64,135 @@ pub(crate) async fn run(args: &Args, config: &Config) -> Result<()> {
         .context("failed to check/run DB migrations")?;
     let conn = db.get().await?;
 
-    let mut dummy_blocks = DummyBlocks::new(args.dummy_blocks, &**conn).await?;
-
+    if args.dummy_blocks {
+        add_dummy_blocks(&mut root, &**conn).await?;
+    }
+    
     info!("Starting to insert realms into the DB...");
-    insert_realm(&**conn, &root, 0, &mut dummy_blocks).await?;
+    insert_realm(&**conn, &root, 0).await?;
     info!("Done inserting realms");
 
     Ok(())
 }
 
+// Wrapper function for recursively adding different blocks.
+fn for_all_realms(realm: &mut Realm, mut f: impl FnMut(&mut Realm)) {
+    fn add_blocks(realm: &mut Realm, f: &mut impl FnMut(&mut Realm)) {
+        f(realm);
+
+        for child in &mut realm.children {
+            add_blocks(child, f);
+        }
+    }
+
+    add_blocks(realm, &mut f);
+}
+
+// This adds different kinds of blocks to empty realms.
```

Comment:
```suggestion
/// Adds different kinds of blocks to empty realms.
```

`///` for comments that are "doc comments" and describe an item  like a function